Demonstrator.md

```diff
@@ -40,17 +38,18 @@ git clone --recurse-submodules https://github.com/kadaster-labs/solid-quest.git
 To start a 'verkoper' and 'koper' POD based on [Community Solid Server](https://github.com/CommunitySolidServer/CommunitySolidServer) use `docker-compose`:
 
 ```bash
-docker compose up solid-pod-provider
+docker compose up --build
 ```
 
 - Verkoper POD: [localhost:3001/verkoper-vera/](http://localhost:3001/verkoper-vera/)
 - Koper POD: [localhost:3001/koper-koos/](http://localhost:3001/koper-koos/)
 
-To start the 'Koopovereenkomst App' switch to the `koopovereenkomst-v3-simple` folder and run `yarn dev`:
+Open the app at [localhost:3000](http://localhost:3000)
 
```

Comment:
Goeie updates!

Zouden we nog extra kunnen / moeten toevoegen dat:

For local development always building the container might not be the fastest development cycle. Therefor a `start-backing-services.sh` script is available to just start the 'backing services' of the app. The app itself can be started with:

```bash
yarn dev
```

Reply:
Hmm... ja, 'vroeger' (lang geleden 😳 ) heb ik wel met makefiles gewerkt ... en is wel een elegante manier om allerlei shortcuts in je project op te nemen. Goeie tip!

Ik ben met je eens dat de 'podman mensen' er zelf wel uitkomen ... en toch helpt zo'n bash file mij om dit soort commands te onthouden 😬 (niet het podman-compose gedeelte, maar het starten van de achtergrond services ...) Als het mij helpt, zou het dan niet ook anderen helpen? 😉 

Blijkbaar heb jij andere skills en systematiek om commands en namen te onthouden?

---

Wat betreft `time` ...

```bash
$ podman system prune
$ time podman-compose build
[...] real    7m3.198s

$ time podman-compose build
[...] real    0m25.435s

$ time podman-compose build --no-cache
[...] real    7m30.040s
```

Works with podman-compose too! 👌

Reply:
Fair enough! Kan zeker handig zijn, heb de documentatie geupdate 👍